Cinebase/Use Cases/Movies/MovieDetail/Showtimes/ShowtimesViewController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ShowtimesViewController.swift
+//  Cinebase
+//
+//  Created by Charles Moncada on 10/05/17.
+//  Copyright © 2017 Globant. All rights reserved.
+//
+
+import UIKit
+
+
+// MARK: ShowtimesView Protocol Definition
+protocol ShowtimesView: class {
+    func handleBuyTicketsButtonPressed(_ sender: UITableViewCell)
```

Comment:
i need to pass the info of Cine and schedule, so what do you suggest? pass the raw info? indexes?